spec/features/takeaway_features_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'order'
+
+feature 'Takeaway ordering system' do
+
+  scenario 'Add multiple items to an order' do
+    myorder = Order.new
+    myorder.add(1, 2)
+    expect(myorder.order[0][3]).to eq 19.0
+  end
+
+  scenario 'calculate total for multiple items' do
+    myorder = Order.new
+    myorder.add(1, 2)
+    myorder.add(1, 2)
+    expect(myorder.total).to eq 38.0
+  end
+
+  xscenario 'send SMS when order is complete' do
+    service = double :twilio
+    allow(service).to receive(:new)
+    myorder = Order.new
+    myorder.add(1, 2)
+    expect(myorder.notify(service)).to receive(:new)
```

Comment:
Remember that the expect(...).to receive syntax is used for declaring expectations about what will happen in the future, i.e as a result of the lines of code following on from the expectation. You've not got any lines of code following on from this expectation so I think you're misusing it a bit.